Resources/public/js/views/ez-contenteditview.js

```diff
@@ -210,6 +214,37 @@ YUI.add('ez-contenteditview', function (Y) {
          */
         _isTouch: function () {
             return Y.UA.touchEnabled;
+        },
+
+        /**
+         * Tap event handler on change language button. It fires `changeLanguage` event.
+         *
+         * @method _changeLanguage
+         * @private
+         * @param {EventFacade} e
+         */
+        _changeLanguage: function (e) {
+            e.preventDefault();
+
+            /**
+             * Fired when the change language link was tapped
+             *
+             * @event changeLanguage
+             */
+            this.fire('changeLanguage');
+        },
+
+        /**
+         * Sets language indicator
+         *
+         * @method setLanguageIndicator
+         * @param {String} languageCode
+         */
+        setLanguageIndicator: function (languageCode) {
```

Comment:
It will simplify a bit the test since `set` is a Y.View (even Y.Base) method. While with a custom method like this one, you need to dynamically create a custom View class with the method and then have a real instance of it, then you have to make the view service a bubble target of this view and then fire the event from this view so that `e.target` is a *real* view with the expected method.